test/config/fileserver_config.json

```diff
@@ -0,0 +1,41 @@
+{"os_image":

```

Comment:
Please review PR 4629. it allows you to specify additional config files from the command line.  then, your script check and use fitcfg() to retrieve the parameters in the extra config file.